cabal.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -eu
+
+stack build && stack run -- ls dependencies constraints > "$1"
```

Comment:
These two commands look wrong to me. Should it not just be `stack ls dependencies constraints > "$1"`? Stack does not have to build Stack to list its dependencies.

See my comment below about the naming of the new subcommand.

(`stack run`, by the way, builds an executable and runs it - see https://docs.haskellstack.org/en/stable/GUIDE_advanced/#the-stack-run-command.)

Reply:
`stack ls dependencies` is such a good lightweight command. I've removed the unnecessary `stack build` that preceded it in the script.